onnxruntime/core/optimizer/unsqueeze_elimination.h

```diff
@@ -3,16 +3,21 @@
 
 #pragma once
 
-#include "core/optimizer/graph_transformer.h"
+#include "core/optimizer/rewrite_rule.h"
 
 namespace onnxruntime {
 
-class UnsqueezeElimination : public onnxruntime::GraphTransformer {
+class UnsqueezeElimination : public RewriteRule {
```

Comment:
>UnsqueezeElimination [](start = 6, length = 20)

Current version of UnsqueezeElimination fails with real models on TensorRT...

just as a test - before checkin can you please enable level1 as default level in inference_session.h and make sure both Linux and Windows TensorRT CI are successful

Reply:
As discussed offline, I kept it out of the default rules (as it was before) until we further investigate why it fails.